server/controllers/conversation.js

```diff
@@ -0,0 +1,167 @@
+
+
+const User = require( "../models/User" );
+const asyncHandler = require( "express-async-handler" );
+const Conversation = require( "../models/conversation" );
+const Message = require( "../models/message" );
+const Profile = require( "../models/Profile" );
+
+// @route Post /create
+// @desc create conversation
+// @access Public
+exports.createConversation = asyncHandler( async ( req, res, next ) => {
+  const userId = await User.findById( req.user.id );
+  if ( !userId )
+  {
+    res.status( 401 );
+    throw new Error( "Not authorized" );
+  }
+  const senderId = req.user.id;
+  const { recipientId, text, conversationId } = req.body;
+  if ( conversationId )
+  {
+    const message = await Message.create( { senderId, text, conversationId } );
```

Comment:
Since this controllers is primarily to create a conversation, we wouldn't want to use it just to create a message and return it (let's remove this)

Reply:
removed the message.create